### ISSUE TITLE
How best to promote openseadragon's toolbar over an Annotorious annotation

### ISSUE THREAD

**Issue by meihuisu** [openseadragon/openseadragon#848]
Annotorious uses z-index when making annotations using Openseadragon's Overlay. When an annotation happens to cover up the Openseadragon toolbar, it renders toolbar useless. When I tested the pure overlay in openseadragon, that is not the case.  Short of suppressing the assignment of  the z-index to annotations in Annotorious' source, perhaps there is a workaround solution?

**Comment by iangilman** [comment]
It'll probably require some z-index fiddling...maybe you could add some z-index to the OpenSeadragon toolbar and see if that helps? 

Have you asked the Annotorious folks?

**Comment by meihuisu** [comment]
I have also posted to Annotorious.

Annotorious actually assigned a z-index to Openseadragon canvas and so it does have one. It is just all the later annotation-overlays
are giving increasing higher numbered z-index. Since toolbar is under the osd,t inherit its z-index, it is masked for sure.  I
have tried playing with some css styling but nothing help, short of removing the z-index off those annotations which worked to unmask
but I am sure that is not the right way. 

I ended up move those button outside of the viewer to avoid the conflict for now.

I will try to find where that built-in toolbar is made and explicitly set its z-index when I get a chance.

----- Original Message -----
